DISDRODB/Raw/EPFL/HPICONET_2010/metadata/32.yml

```diff
@@ -9,22 +9,22 @@ source: ''
 source_convention: ''
 source_processing_date: ''
 title: HPICONET_2010
-description: ''
+description: 'Pre-campaign of the HyMeX (https://hymex.org/) experiment '
 project_name: HPICONET_2010
 keywords: ''
 summary: ''
 history: ''
-comment: ''
+comment: 'Location of the instrument reconstructed indirectly some years apart. Location error may be possible. '
 station_id: '32'
-location: ''
-country: ''
-continent: ''
-latitude: -9999
-longitude: -9999
-altitude: -9999
-deployment_status: ''
+location: "Ard\xE8che"
```

Comment:
Looks bad. Maybe use e instad of è

Reply:
Era già cosi nel file creato prima che ci mettessi le mani io in realtà